test/behandlerdialog/MeldingTilBehandlerTest.tsx

```diff
@@ -48,7 +67,49 @@ describe("MeldingTilBehandler", () => {
       )
     ).to.exist;
   });
+
+  const selectLabel = "Hvilken meldingstype ønsker du å sende";
+
   describe("MeldingTilBehandlerSkjema", () => {
+    it("Viser select komponent for valg av meldingstype", () => {
+      renderMeldingTilBehandler();
+
+      expect(screen.getByLabelText(selectLabel)).to.exist;
+      fireEvent.change(screen.getByLabelText(selectLabel), {
+        target: { value: MeldingType.FORESPORSEL_PASIENT_TILLEGGSOPPLYSNINGER },
+      });
+      const selectElement: HTMLSelectElement =
+        screen.getByLabelText(selectLabel);
+      expect(selectElement.value).to.equal(
+        MeldingType.FORESPORSEL_PASIENT_TILLEGGSOPPLYSNINGER
+      );
+      expect(selectElement.value).to.not.equal(
+        MeldingType.FORESPORSEL_PASIENT_LEGEERKLARING
+      );
+
+      fireEvent.change(screen.getByLabelText(selectLabel), {
+        target: { value: MeldingType.FORESPORSEL_PASIENT_LEGEERKLARING },
+      });
+      expect(selectElement.value).to.equal(
+        MeldingType.FORESPORSEL_PASIENT_LEGEERKLARING
+      );
+      expect(selectElement.value).to.not.equal(
+        MeldingType.FORESPORSEL_PASIENT_TILLEGGSOPPLYSNINGER
+      );
```

Comment:
Kan gjerne bruke mer spacing/linjeavsnitt her for å indikere de ulike stegene i testen 👍🏼